test/test_other.py

```diff
@@ -13270,7 +13270,7 @@ def test_min_node_version(self):
     node_version = shared.check_node_version()
     node_version = '.'.join(str(x) for x in node_version)
     self.set_setting('MIN_NODE_VERSION', 210000)
-    expected = 'This emscripten-generated code requires node v21.0.0 (detected v%s)' % node_version
+    expected = 'This emscripten-generated code requires node v21.0.0 (detected v%s' % node_version
```

Comment:
is this intentional?

Reply:
Yes, on my machine (and with any canary build of node) the real error looks something like this:

`This emscripten-generated code requires node v21.0.0 (detected v20.0.0-v8-canary-xxx)`

This seemed like the easiest way to make the error match.

Reply:
but why remove the closing paren?

Reply:
Oh because `shared.check_node_version()` returns just `20,0,0` and not the extra stuff that is in the error message.

This we we are effectively matching any version that starts with `v20.0.0`

Reply:
I guess we could try to be more precise but I'm not sure its worth it?

Reply:
yeah I think this is fine.